go.mod

```diff
@@ -96,8 +95,6 @@ require (
 	github.com/hashicorp/packer-plugin-vmware v1.0.7
 	github.com/hashicorp/packer-plugin-vsphere v1.0.5
 	github.com/hashicorp/packer-plugin-yandex v1.1.1
-	github.com/prometheus/common v0.9.1
```

Comment:
That one looks off, is this a transitive dependency that's removed?

Reply:
This is a side effect of running `go mod tidy` it is possible that it came in with the recent hcp-sdk-go changes.